xarray/core/accessors.py

```diff
@@ -258,3 +258,24 @@ def round(self, freq):
             Array-like of datetime fields accessed for each element in values
         '''
         return self._tslib_round_accessor("round", freq)
+
+    def strftime(self, date_format):
+        values = self._obj.data
+        values_as_series = pd.Series(values.ravel())
+        strs = values_as_series.dt.strftime(date_format)
+        return strs.values.reshape(values.shape)
```

Comment:
For consistency with the other time methods, it would be good to:
1. support dask arrays
2. return an `xarray.DataArray`